content/release-10-18+/streaming-analytics-10-18+-bundle/10.18+_0.md

```diff
@@ -53,6 +53,11 @@ You can now create a selection list for an Analytics Builder template parameter.
 You can define selection lists for types such as string, float, source or destination, or geofence and you can also select a specific value to be the default value. The values that you define for a selection list are then available for selection when you create instances of the model.
 See also [Adding a selection list for a template parameter](https://cumulocity.com/docs/streaming-analytics/analytics-builder/#adding-a-selection-list-for-a-template-parameter) in the user documentation.
 
+#### Missing subassets in a group
```

Comment:
This is also true for assets and not just device group. If a device/asset is missing from an asset hierarchy we no longer throw an error. Does saying "subasset of a group " cover the assets too?